tmt/tests/system-reinstall/test-system-reinstall-bootc.sh

```diff
@@ -0,0 +1,140 @@
+#!/bin/bash
+set -exuo pipefail
+
+# Check environment
+printenv
+
+# This script only runs on Packit and gating environment
+if [ "$TMT_REBOOT_COUNT" -eq 0 ]; then
+    pwd
+    # change to hack directory
+    cd ../../hack
+
+    ARCH=$(uname -m)
+    # Get OS info
+    source /etc/os-release
+    case "$ID" in
+        "centos")
+            BASE="quay.io/centos-bootc/centos-bootc:stream${VERSION_ID}"
+            TEST_OS="${ID}-stream-${VERSION_ID}"
+            # Some rhts-*, rstrnt-* and tmt-* commands are in /usr/local/bin
+            cp -r /usr/local/bin .
+            ;;
+        "fedora")
+            BASE="quay.io/fedora/fedora-bootc:${VERSION_ID}"
+            TEST_OS="${ID}-${VERSION_ID}"
+            if [[ "$VERSION_ID" == 44 ]]; then
+                TEST_OS="${ID}-rawhide"
+            fi
+            # Some rhts-*, rstrnt-* and tmt-* commands are in /usr/local/bin
+            cp -r /usr/local/bin .
+            ;;
+        "rhel")
+            # OSCI gating only
+            BASE="images.paas.redhat.com/bootc/rhel-bootc:latest-${VERSION_ID}"
+            # Some rhts-*, rstrnt-* and tmt-* commands are in /var/lib/tmt/scripts
```

Comment:
Why duplicate this code in all three cases instead of having it be a second step?

Reply:
That's wired thing. The Packit, Fedora CI and  testing farm keep useing `/usr/local/bin` to save `tmt-` files, but OSCI changed to save `tmt-` files into `/var/lib/tmt/scripts`. So I have to make them separated. `rhel` means OSCI in this case.